src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -383,16 +394,22 @@ add_slope_flags <- function(df) {
 # }
 ```
 
+add large anomaly flag function
+```{r}
+add_large_anomaly_flag <- function(df) {
+  df <- df %>% 
+    mutate(flag_binary = ifelse((is.na(flag) | flag == "24hr anomaly flag"), 0, 1),
+           roll_bin = data.table::frollsum(flag_binary, n = 97, align = 'center', na.rm = F, fill = NA_real_)) %>% 
```

Comment:
```suggestion
           # create a rolling count of flags over the 12-h prior and 12-h after the current row of data
           roll_bin = frollsum(flag_binary, n = 97, align = 'center', na.rm = F, fill = NA_real_)) %>% 
```
no need to use package name here, no conflicting function names - in an effort to make more literate code, I'd also suggest renaming this to something that helps an outsider know what this column is, for instance 'rolling_flag_count' or the like.